streaming/java/streaming-runtime/src/main/java/org/ray/streaming/runtime/core/graph/executiongraph/ExecutionJobVertex.java

```diff
@@ -0,0 +1,174 @@
+package org.ray.streaming.runtime.core.graph.executiongraph;
+
+import com.google.common.base.Preconditions;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.aeonbits.owner.ConfigCache;
+import org.ray.api.RayActor;
+import org.ray.streaming.jobgraph.JobVertex;
+import org.ray.streaming.jobgraph.LanguageType;
+import org.ray.streaming.jobgraph.VertexType;
+import org.ray.streaming.operator.StreamOperator;
+import org.ray.streaming.runtime.config.master.ResourceConfig;
+import org.ray.streaming.runtime.core.processor.ProcessBuilder;
+import org.ray.streaming.runtime.core.processor.StreamProcessor;
+import org.ray.streaming.runtime.worker.JobWorker;
+
+/**
+ * Physical job vertex which including parallelism execution vertex.
+ */
+public class ExecutionJobVertex {
+
+  private final String jobName;
+  private final int jobVertexId;
+  private final String jobVertexName;
+  private final JobVertex jobVertex;
+  private final StreamProcessor streamProcessor;
+  private int parallelism;
+  private Map<String, String> jobConfig;
+  private Map<String, Double> resources;
+  private long buildTime;
+  private List<ExecutionVertex> executionVertexList;
+
+  private List<ExecutionJobEdge> inputEdges = new ArrayList<>();
+  private List<ExecutionJobEdge> outputEdges = new ArrayList<>();
+
+  public ExecutionJobVertex(String jobName, JobVertex jobVertex, Map<String, String> jobConfig,
+      long buildTime) {
+    this.jobName = jobName;
+    this.jobVertexId = jobVertex.getVertexId();
+    this.jobVertexName = generateVertexName(jobVertexId, jobVertex.getStreamOperator());
+    this.jobVertex = jobVertex;
+    this.streamProcessor = ProcessBuilder.buildProcessor(jobVertex.getStreamOperator());
+    this.parallelism = jobVertex.getParallelism();
+    this.jobConfig = jobConfig;
+    this.resources = generateResources();
+    this.buildTime = buildTime;
+    this.executionVertexList = createExecutionVertics();
+  }
+
+  private String generateVertexName(int vertexId, StreamOperator streamOperator) {
+    return vertexId + "-" + streamOperator.getName();
+  }
+
+  private Map<String, Double> generateResources() {
```

Comment:
`Resource` is not related to this PR?

Reply:
Removed this. Please restore this in next PR. @datayjz